.codeclimate.yml

```diff
@@ -0,0 +1,13 @@
+engines:
+  duplication:
+    enabled: false
+  fixme:
+    enabled: true
+  radon:
+    enabled: true
```

Comment:
Note that none of these are actually doing security scanning...

/cc https://github.com/18F/compliance-toolkit/issues/14